doc/cephadm/services/osd.rst

```diff
@@ -924,6 +924,57 @@ It is also possible to specify directly device paths in specific hosts like the
 
 This can easily be done with other filters, like `size` or `vendor` as well.
 
+It's possible to specify the `crush_device_class` parameter within the
+DriveGroup spec, and it's applied to all the devices defined by the `paths`
+keyword:
+
+.. code-block:: yaml
+
+    service_type: osd
+    service_id: osd_using_paths
+    placement:
+      hosts:
+        - Node01
+        - Node02
+    crush_device_class: ssd
+    spec:
+      data_devices:
+        paths:
+        - /dev/sdb
+        - /dev/sdc
+      db_devices:
+        paths:
+        - /dev/sdd
+      wal_devices:
+        paths:
+        - /dev/sde
+
+The `crush_device_class` parameter, however, can be defined for each OSD passed
+using the `paths` keyword with the following syntax:
+
+.. code-block:: yaml
+
+    service_type: osd
+    service_id: osd_using_paths
+    placement:
+      hosts:
+        - Node01
+        - Node02
+    crush_device_class: ssd
+    spec:
+      data_devices:
+        paths:
+        - data: /dev/sdb
+        - crush_device_class: ssd
+        - data: /dev/sdc
+        - crush_device_class: nvme
```

Comment:
Is this the right YAML structure? I expected something more like:
```
   data_devices:
        paths:
        - data: /dev/sdb
             crush_device_class: ssd
        - data: /dev/sdc
            crush_device_class: nvme
```

Reply:
You're right, I had an issue in the previous change. The right structure is the following:

```
    ...
    ...
    spec:
      data_devices:
        paths:
        - data: /dev/sdb
          crush_device_class: ssd
        - data: /dev/sdc
          crush_device_class: nvme
      db_devices:
         ... 
```
where paths is translated into a json like `paths: [ { "data":  "/dev/sdb", "crush_device_class": "ssd" }, { "data":  "/dev/sdc", "crush_device_class": "nvme" } ] ` which is easy to process (and validate) from cephadm.
Thanks, nice catch on that issue I had with the yaml definition, just updated the doc change!

Reply:
YAML!  I hate YAML! Even with strawberries!
;)